download/download.go

```diff
@@ -13,7 +13,7 @@ import (
 
 func DownloadAll() {
 	download("data/source/languages.json", "https://raw.githubusercontent.com/bdswiss/country-language/master/data.json")
-	download("data/source/countries.csv", "https://raw.githubusercontent.com/datasets/country-codes/master/data/country-codes.csv")
+	download("data/source/countries.csv", "https://raw.githubusercontent.com/datasets/country-codes/2ed03b6993e817845c504ce9626d519376c8acaa/data/country-codes.csv")
```

Comment:
I forgot to commit this last year: https://github.com/flowcommerce/json-reference/pull/115#issuecomment-1165791872
The reason we need to do this is outlined here: https://github.com/flowcommerce/json-reference/pull/116#issue-1284010752